### PR TITLE
Allow duration of temporary address change to be seven days

### LINKED ISSUE #411: Allow duration of temporary address change to be seven days
In Mitt Konto, the minimum duration for a temporary address change for a subscription is supposed to be seven days, but due to a small mistake in our code, it is actually eight days. Here is the incriminated code:

https://github.com/KSF-Media/affresco/blob/7fd4f520db90368668e3ae1fc343052da0be8dec/packages/components/src/TemporaryAddressChange/Component.purs#L98-L100

We should fix this so that the minimum duration can actually be seven days, because the way we do it we add seven days to the starting date, which makes a total of eight days.

### PR DESCRIPTION
**Pull request by mrskug**
Fixes #411

### CODE REVIEW

File: packages/components/src/TemporaryAddressChange/Component.purs [hunk 1]
```diff
@@ -96,7 +96,7 @@ component = React.createComponent "TemporaryAddressChange"
 calcMinEndDate :: Maybe DateTime -> Maybe DateTime
 calcMinEndDate Nothing = Nothing
 calcMinEndDate (Just startDate) = do
-  let week = Time.Duration.Days 7.0
+  let week = Time.Duration.Days 6.0
```
Comment (ae-mo) [inline packages/components/src/TemporaryAddressChange/Component.purs:99]: Technically this is not a week anymore 😆 maybe you can call it something like `interval` or `days` (I don't  really know) and add a comment on why it's `6.0`.

Comment (mrskug) [reply packages/components/src/TemporaryAddressChange/Component.purs:99]: I think `week` is a good name here as that is the result of the action we are doing here and conveys the intent better than just `interval` or `days`

Comment (ae-mo) [reply packages/components/src/TemporaryAddressChange/Component.purs:99]: I'd rather add a comment to explain that we want the minimum duration to be one week

Comment (pikkuruusu) [reply packages/components/src/TemporaryAddressChange/Component.purs:99]: So, could we find a solution here? Customer service is anxious.

Comment (mrskug) [reply packages/components/src/TemporaryAddressChange/Component.purs:100]: Feel free to merge at any time

**Review by ae-mo — APPROVED** [review]

**Pull request merged**